Resources/input/fsh/ISiKCapabilityStatementDokumentenaustauschServer.fsh

```diff
@@ -16,80 +32,121 @@ Usage: #definition
 * rest.mode = #server
   
 * rest.resource[+]
-  * extension.url = "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation"
-  * extension.valueCode = #SHALL
+  * insert Expectation (#SHALL)

```

Comment:
todo: bitte hier jemand anderes für das review heranziehen - @patrick-werner  übernimmst du das? (Ich bin nicht in der Syntax bewandert).